monri/build.gradle

```diff
@@ -88,6 +88,7 @@ dependencies {
     testImplementation "junit:junit:${rootProject.ext.junitVersion}"
     testImplementation "org.mockito:mockito-core:${rootProject.ext.mockitoCoreVersion}"
     testImplementation "org.robolectric:robolectric:${rootProject.ext.robolectricVersion}"
+    testImplementation "org.json:json:${rootProject.ext.json}"
```

Comment:
@adnanomerovic Is there platform supported JSON api? This [one](https://developer.android.com/reference/org/json/JSONObject) maybe? Or it's the same as the one you've included?

Reply:
That's for testing, without that I get exception when I'm try to  create new JSONObject

Reply:
I completely missed `testImplementation` part. Good one! 

Without this it would be impossible to test model serialize/deserialize without having test moved to androidTest (in that case we have JSONObject support)?

Reply:
yea "new JSONObject("jsonString")" won't work without that lib (in android test)

Reply:
Right now I didn't write too much test, but later probably we will needed (I guess)

Reply:
We'll cover test cases in a separate issue. It's important topic and should be addressed accordingly. 

Follow [issue](https://github.com/MonriPayments/monri-android/issues/16) for updates.